index.html

```diff
@@ -82,402 +83,384 @@
+              </button>
+            </div>
+          </li>
+        </ul>
+      </div>
+    </section>
 
-            <ul class="tab-list" data-reveal="left">
-              <li>
-                <button class="tab-btn active">Lorem</button>
-              </li>
+    <section class="about" aria-labelledby="about-label">
+      <div class="about__container container">
+        <div class="about__content">
+          <p class="section-subtitle title-lg has-after" id="about-label" data-reveal="left">
+            O mnie
+          </p>
 
-              <li>
-                <button class="tab-btn">Ipsum</button>
-              </li>
+          <h2 class="headline-md" data-reveal="left">
+            Doswiadczenie zawodowe
+          </h2>
 
-              <li>
-                <button class="tab-btn">dr Nowak</button>
-              </li>
-            </ul>
+          <p class="about__section-text" data-reveal="left">
+            Setki wyleczonych pacjentów, doswiadczenie zawodowe, ktore pozwala na wykonanie nawet najtrudniejszych
+            zabiegow.
+          </p>
 
-            <p class="tab-text" data-reveal="left">
-              Aliquam faucibus, odio nec commodo aliquam, neque felis placerat
-              dui, a porta ante lectus dapibus est. Aliquam a bibendum mi, sed
-              condimentum
-            </p>
+          <ul class="about__tab-buttons-list" data-reveal="left">
+            <li>
+              <button class="about__tab-btn about__tab-btn-active">Lorem</button>
+            </li>
 
-            <div class="wrapper">
-              <ul class="about-list">
-                <li class="about-item" data-reveal="left">
-                  <ion-icon name="checkmark-circle-outline"></ion-icon>
+            <li>
+              <button class="about__tab-btn">Ipsum</button>
+            </li>
 
-                  <span class="span">Sonsectetur adipisicing elit</span>
-                </li>
+            <li>
+              <button class="about__tab-btn">dr Nowak</button>
+            </li>
+          </ul>
 
-                <li class="about-item" data-reveal="left">
-                  <ion-icon name="checkmark-circle-outline"></ion-icon>
+          <p class="tab-text" data-reveal="left">
+            Aliquam faucibus, odio nec commodo aliquam, neque felis placerat
+            dui, a porta ante lectus dapibus est. Aliquam a bibendum mi, sed
+            condimentum
+          </p>
 
-                  <span class="span">Exercitation ullamco laboris</span>
-                </li>
+          <div class="wrapper">
+            <ul class="about__tab-list">
+              <li class="about__item" data-reveal="left">
+                <ion-icon name="checkmark-circle-outline"></ion-icon>
 
-                <li class="about-item" data-reveal="left">
-                  <ion-icon name="checkmark-circle-outline"></ion-icon>
+                <span class="span">Sonsectetur adipisicing elit</span>
```

Comment:
That's a great name for a class xD

Reply:
That class do nothing, forget to delete :)